int64 grasp_id
string object_name
geometry_msgs/PoseStamped preshape_palm_world_pose
geometry_msgs/PoseStamped true_preshape_palm_world_pose
sensor_msgs/JointState preshape_allegro_joint_state
sensor_msgs/JointState true_preshape_joint_state
int32 grasp_success_label
bool top_grasp
int32[] sparse_voxel_grid
float32[] object_size
---
bool success
int64 object_id
